test-case-reporting/src/test_result_record.ts

```diff
@@ -304,9 +304,13 @@ export class TestResultRecord {
    */
   async getTestCasesSortedByStatColumn(
     {limit, offset}: PageInfo,
-    column: 'passed' | 'failed' | 'skipped' | 'errored',
+    column: string,
     sampleSize: number
   ): Promise<Array<TestCase>> {
+    if (!['passed', 'failed', 'skipped', 'errored'].includes(column)) {
+      throw new TypeError('Bad column used for sorting test cases');
```

Comment:
```suggestion
      throw new TypeError(`Bad column used for sorting test cases: "${column}"`);
```

Reply:
Done, though I renamed 'column' to 'stat'; let me know if you disagree and I'll revert it.